### ISSUE TITLE
Angular 2-way binding not working in 3.7.0

### ISSUE THREAD

**Issue by jainpradeep** [jspreadsheet/ce#821]
Sir,

Recent update in the component just broke 2 way binding in Angular 8 (TypeScript). Jexcel cell view values are not reflecting in angular components. It is working with v3.4.4. Request to please look into the issue.

**Comment by pphod** [comment]
If you want some support on this my skype is paul.hodel

We need to make sure this is not something on the angular side since I am not aware of any changes would affect that.

**Comment by jainpradeep** [comment]
Sir,

I added a save button in your example for angular to reproduce the issue. 

Data from the component is populating in JExcel grid but if we update the Jexcel grid component data (logged in alert on save click) is not getting updated.

This is happening in jExcel 3.7+ versions. 2 way binding works well with older version. I have verified with the same example in v3.4.

[https://codesandbox.io/s/jexcel-and-angular-hrbpl](https://codesandbox.io/s/jexcel-and-angular-hrbpl)
